native/native_image.go

```diff
@@ -38,33 +38,44 @@ type NativeImage struct {
 	Arguments       []string
 	Executor        effect.Executor
 	Logger          bard.Logger
-	Manifest        *properties.Properties
+	nativeMain 		nativeMain
 	StackID         string
 	Compressor      string
 }
 
-func NewNativeImage(applicationPath string, arguments string, compressor string, manifest *properties.Properties, stackID string) (NativeImage, error) {
+func NewNativeImage(applicationPath string, arguments string, compressor string, manifest *properties.Properties, stackID string, ops ...NativeImageOption) (NativeImage, error) {
```

Comment:
I'm OK with using the "ops" approach, but I would like to see it used consistently, by which I mean basically exposing the other arguments to NewNativeImage in the same way. I know it's not really related to this change, but I think it would be appropriate refactor to include if we want to switch to using this style of passing in arguments.

Alternatively, stick with the static argument list and if something isn't required just stick with nil or the default value for that item.

Reply:
I kept the static argument strategy just because it was only 1, but could be great to do the refactoring